### DISCUSSION TITLE
PWA Add to home screen only brings up manifest on Android

### DISCUSSION THREAD

**Discussion by RonV42** [poblabs/weewx-belchertown#749] [Q&A]
I have had this skin up and running since just before Christmas and I would say it's great.  One thing that I cannot get working is on my Android device selecting "Add to Phone" or "Add to Home Screen".  When using that method and clicking on the icon all that displays is the json manifest for the PWA.  The app never loads.  Anyone else seeing this?  I am attaching a screen shot, I have tried in both the EDGE and Chrome browsers on the phone.   When I save it as a "APP" on Windows with EDGE it works just fine,  I am stumped.

If you want to check out my weather it's at:  http://weather.vargofamily.com

![Screenshot_20220102-132502](https://user-images.githubusercontent.com/34961476/147887287-736a7ec4-c7a9-4ddf-b6df-79c9a62f32a8.png)
.

**Comment by averagejoej** [comment]
FWIW, iPhone worked fine on your site.  Is there a way you can sniff http traffic thru something like fiddler on a desktop?  Not an android person, but I know this is pretty easy on iOS.

**Reply by RonV42** [reply]
That is what I was worred about the PWA manifests or some security aspect is not compatible with Android.  I have tried on 4 different phones, Samsung, TCL, Motorola, and some off brand and all have the same issue when pinned to start they only display the manifest and don't launch the app.

**Reply by herrfrei** [reply]
@RonV42 I got it working on Android with changing `start_url` from empty string to `./index.html`:

```json
"start_url": "./index.html"
```
